### DISCUSSION TITLE
Chat Sync Issue between PC and Mobile Application

### DISCUSSION THREAD

**Discussion by karpalig** [ChatGPTNextWeb/NextChat#4384] [Q&A] [answered]
While the application runs smoothly both on my PC and mobile device, I have noticed that the chats are not synchronizing between the two platforms. This is causing me inconvenience as I frequently switch between devices and need to keep track of my conversations.

I would appreciate it if you could assist me in resolving this issue. Could you please provide some guidance or troubleshooting steps to ensure that the chat history is synchronized across all devices?

For your reference,
iOS chrome and PC chrome are last for today

**Comment by karpalig** [comment]
Can I establish a connection between two devices if I'm using one API key? Any suggestions?

**Reply by 916980name** [reply, marked as answer]
The data stores in browser local storage, can not synchronize so far. This is a simple user interface, not a full-function chat app which contains account manage, persistent storage or any others. Export functionality is now available, maybe you could request for a data import feature.

**Comment by karpalig** [comment]
I have successfully connected to the WebDAV (Selfhost) and received confirmation that the synchronization is complete. However, the files are not appearing on the server. Should any from chats/messages/prompts/masks be synced?

<img width="946" height="154" alt="image" src="https://github.com/user-attachments/assets/81305e65-c360-45bc-b8ea-6c36fe18feb2" />

Thank you for your support.

Best regards,
